src/useObservable.ts

```diff
@@ -3,9 +3,17 @@ import { useRef } from "react"
 
 type SupportedValues = object | Map<any, any> | any[]
 
-export function useObservable<T extends SupportedValues>(initialValue: T): T {
+// The two overloads below seems strange. But if I combine them to `initialValue: (() => T) | T)`,
+// typescript would infer a wrong return type. tslint tells me to combine them, so just ignore tslint.
+export function useObservable<T extends SupportedValues>(initialValue: () => T): T
+// tslint:disable-next-line

```

Comment:
Let's be specific on which rule is wrong
```suggestion
// tslint:disable-next-line unified-signatures
```

Reply:
fixed